integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -413,17 +414,20 @@ void testDomainK8sEventsScalePastMaxWithoutChangingIntrospectVersion() {
   void testDomainK8sEventsScalePastMaxAndChangeIntrospectVersion() {
     OffsetDateTime timestamp = now();
     try {
-      logger.info("Scaling cluster using patching");
+      removeReplicasSettingAndVerify(domainUid, cluster1Name, domainNamespace3, replicaCount,
+          managedServerPodNamePrefix);
+
       String introspectVersion = assertDoesNotThrow(() -> getNextIntrospectVersion(domainUid, domainNamespace3));
-      assertFalse(scaleCluster(cluster1Name, domainNamespace3, 3), "failed to scale cluster via patching");
       String patchStr
           = "["
-          + "{\"op\": \"replace\", \"path\": \"/spec/introspectVersion\", \"value\": \"" + introspectVersion + "\"}"
+          + "{\"op\": \"replace\", \"path\": \"/spec/introspectVersion\", \"value\": \"" + introspectVersion + "\"},"
+          + "{\"op\": \"replace\", \"path\": \"/spec/replicas\", \"value\": 3}"
           + "]";
```

Comment:
pass value from introspectVersion instead of 3

Reply:
Hmmm, the value 3 is the domain's new replica count. The introspectVersion value is from the variable introspectVersion. Or I might have not understood your comment correctly.